templates/assets.html

```diff
@@ -13,6 +13,11 @@
 
 {% block page_content %}
     <div class="assets">
+        <div class="assets-intro">
+            A collection of Bevy assets, plugins, learning resources, and apps made by the community. If you would like to
```

Comment:
```suggestion
            A collection of third-party Bevy assets, plugins, learning resources, and apps submitted by community members. If you would like to
```

Reply:
I tried to not change the wording from the README to be as not controversial as possible. I don't mind the "third party", but I'm not sure what's wrong with "made by the community".

Reply:
Agreed, 'made by the community' rolls off the tongue much more easily. In addition, the next sentence already contains the word 'submit', so now you're using the same word twice in short succession.